CRM/Upgrade/Incremental/sql/5.7.alpha1.mysql.tpl

```diff
@@ -6,7 +6,7 @@ UPDATE civicrm_navigation SET icon = "crm-i fa-address-book-o" WHERE name = "Con
 UPDATE civicrm_navigation SET icon = "crm-i fa-money" WHERE name = "Contributions" AND domain_id = {$domainID} AND icon IS NULL;
 UPDATE civicrm_navigation SET icon = "crm-i fa-calendar" WHERE name = "Events" AND domain_id = {$domainID} AND icon IS NULL;
 UPDATE civicrm_navigation SET icon = "crm-i fa-envelope-o" WHERE name = "Mailings" AND domain_id = {$domainID} AND icon IS NULL;
-UPDATE civicrm_navigation SET icon = "crm-i fa-users" WHERE name = "Memberships" AND domain_id = {$domainID} AND icon IS NULL;
+UPDATE civicrm_navigation SET icon = "crm-i fa-id-badge" WHERE name = "Memberships" AND domain_id = {$domainID} AND icon IS NULL;
```

Comment:
@colemanw I think we will need an upgrade step in 5.7.beta1.mysql.tpl for anyone has already upgraded to the rc